### PR TITLE
docs: Typo in space-infix-ops docs 

### LINKED ISSUE #15736: Typo in space-infix-ops docs
### URL(s)

https://eslint.org/docs/rules/space-infix-ops#space-infix-ops

### What did you do?

I was reading documentation...

The issue I am reporting has to do with the **ESLint Rule:** **_`space-infix-ops`_**

### The "rules/space-infix-ops" page contains the following line:

> ##### "The proponents of these extra spaces **[believe it make](https://eslint.org/docs/rules/space-infix-ops#space-infix-ops)** the code easier to read and can more easily highlight potential errors, such as:"

### The line should read...

> ##### "Proponents of this rule believe that it makes code easier to read."

### Participation

- [X] I am willing to submit a pull request for this issue.

### Additional comments

I want to submit a pull request to fix the documentation for this page, but I want to make sure that I am requesting, and pulling the correct repository. My email is W3Dojo@Gmail.com, and I would be happy to fix this issue my self, and maybe even some others, if you could send me the docs, or what have you on the standard procedure for requesting a pull request for an ESLint/ESLint-website issue.

### PR DESCRIPTION
**Pull request by kmin-jeong**
Fixes #15736 

#### Prerequisites checklist

- [v] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
Fix a misspelling and Typo in space-infix-ops docs

#### Is there anything you'd like reviewers to focus on?
I wish revierwers check to see if it has been modified to fit the context.

### CODE REVIEW

**Review by snitin315 — APPROVED** [review]
LGTM. Let's wait for another review before merging.

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks for contributing!

**Pull request merged**
